avro/profile.json

```diff
@@ -0,0 +1,118 @@
+{
+  "name":"profile",
+  "type":"record",
+  "namespace": "dto",
+  "doc":"Schema for cosmos api container 'profiles'",
+  "fields":[
+    {
+      "name": "id",
+      "type": "string",
+      "default": "undefined"
```

Comment:
IMHO this field is required (all the document have it and should be a combination of `userPDVId` and `version`)

Reply:
required field done in https://github.com/pagopa/io-functions-elt/pull/81/commits/9c449a0343484f07576dcfbc160d613e0651c285